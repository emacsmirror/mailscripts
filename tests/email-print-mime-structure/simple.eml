Date: Sun, 24 Nov 2019 21:13:45 -0500
Subject: test message
Message-ID: <simple@mailscripts.example>
From: Alice <alice@mailscripts.example>
To: Bob <bob@mailscripts.example>

This is a simple message


