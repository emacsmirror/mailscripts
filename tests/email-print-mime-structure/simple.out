└─╴text/plain 30 bytes
